components/button/test/button-add.axe.js

```diff
@@ -0,0 +1,40 @@
+import '../button-add.js';
+import { expect, fixture, html, oneEvent } from '@brightspace-ui/testing';
+
+describe('d2l-button-add', () => {
+
+	it('default', async() => {
+		const el = await fixture(html`<d2l-button-add></d2l-button-add>`);
+		await expect(el).to.be.accessible();
+	});
+
+	it('label', async() => {
+		const el = await fixture(html`<d2l-button-add label="Custom Label"></d2l-button-add>`);
+		await expect(el).to.be.accessible();
+	});
+
+	it('text', async() => {
+		const el = await fixture(html`<d2l-button-add text="Custom Text"></d2l-button-add>`);
+		await expect(el).to.be.accessible();
+	});
+
+	it('visible text', async() => {
+		const el = await fixture(html`<d2l-button-add visible-text></d2l-button-add>`);
+		await expect(el).to.be.accessible();
+	});
+
+	it('focused', async() => {
+		const el = await fixture(html`<d2l-button-add></d2l-button-add>`);
+		setTimeout(() => el.focus());
+		await oneEvent(el, 'focus');
+		await expect(el).to.be.accessible();
```

Comment:
This uses the actual keyboard instead of the JS API which won't trigger `:focus-visible`.

```suggestion
		const el = await fixture(html`<d2l-button-add></d2l-button-add>`);
		await focusElem(el);
		await expect(el).to.be.accessible();
```